src/evict/evict_page.c

```diff
@@ -728,10 +728,8 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
     if (ret != 0)
         WT_STAT_CONN_INCR(session, cache_eviction_fail_in_reconciliation);
 
-    if (snapshot_acquired) {
-        --session->txn->forced_iso;
-        __wt_txn_release_snapshot(session);
-    }
+    if (restore_snapshot)
```

Comment:
I liked the "snapshot_acquired" name better. It signals what has happened rather than what we need to do. Feel free to ignore me on this one if you disagree.

Reply:
Agreed. Fixed in c05646f.